tests/files/env_config.yml

```diff
@@ -0,0 +1,9 @@
+environments:
+  development:
+    prefix: Dev-Prefix-
+
+  staging:
+    prefix: Stage-Prefix-
+
+  production:
+    prefix: Prod-Prefix-
```

Comment:
Having this config file seems necessary because of the way you're implementing environments though, no? I'm having trouble seeing how it's orthogonal.

If this is an intermediate path that allows for an easier transition, that might make sense, but unless I'm misunderstanding something here it actually looks a little more difficult. The transition to what I suggested below could just be moving `us-east-1.yml` to `environments/sandbox.yml` and adding the region as a parameter at the top.

Though—maybe it's the parsing and interpretation of that info by stacker that's more complex? You've worked with this more than I have.